test/functional/rpc_getblockstats.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
```

Comment:
What are the extra three lines added contrary to the initial pull

Reply:
Good question :) That's because I converted the test to use mocktime and not realtime when generating blocks for the test thus I needed https://github.com/dashpay/dash/pull/3058/files/0aa34ea89cf3630342e9ed6aa49291652cdccc83#diff-dbccfb6d2ff211cd44b8f5e532e3238aR13 and also, we have no `set_test_params` functionality yet so had to convert it to `def __init__(self):` + `super().__init__()` https://github.com/dashpay/dash/pull/3058/files/0aa34ea89cf3630342e9ed6aa49291652cdccc83#diff-dbccfb6d2ff211cd44b8f5e532e3238aR47.

`get_mocktime` is deprecated after #3049 however, so I rebased it (with no changes) and added 3a01dfb5b on top.